test-data/unit/check-namedtuple.test

```diff
@@ -853,3 +853,10 @@ b = (1, 2)
 if not b:
     ''()  # E: "str" not callable
 [builtins fixtures/tuple.pyi]
+
+[case testNamedTupleAtRunTime]
+from typing import NamedTuple
+
+class A:
+    def __init__(self) -> None:
+        self.b = NamedTuple('x', [('s', str), ('n', int)])  # E: NamedTuple type as an attribute is not supported
```

Comment:
As I mentioned above, it would make sense to move this to `pythoneval.test`, since the behavior may depend on how `NamedTuple` is defined in stubs, and typeshed has a different definition than the simplified stubs used in most tests.

Also, add something like `reveal_type(A().b)  # N: ...` to the end of the test case.